core/citrus-spring/src/test/java/org/citrusframework/reporter/ReporterConfigDisabledTest.java

```diff
@@ -0,0 +1,32 @@
+package org.citrusframework.reporter;
+
+import static org.citrusframework.reporter.ReporterConfig.CITRUS_HTML_REPORTER;
+import static org.citrusframework.reporter.ReporterConfig.CITRUS_JUNIT_REPORTER;
+import static org.citrusframework.reporter.ReporterConfig.CITRUS_LOGGING_REPORTER;
+
+import org.citrusframework.UnitTestSupport;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.test.context.TestPropertySource;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+@TestPropertySource(value = "classpath:org/citrusframework/reporter/default-logger-disabled.properties")
+public class ReporterConfigDisabledTest extends UnitTestSupport {
```

Comment:
just a side-node: that is an `*IT` imo, but I don't know if the core is already properly cleaned up. I've done it for the simulator, tho. but no objection at this point.